src/components/TokenPickers/validation.js

```diff
@@ -10,11 +10,11 @@ const offsetValidation = (value, radioValue) => {
   return undefined;
 };
 
-const dateValidation = (value, radioValue, dateMoment, dateFormat) => {
+const dateValidation = (value, radioValue, dateDayjs, dateFormat) => {
   if (radioValue === RADIO_VALUE_DATE && (
     !value ||
-    !dateMoment?._isValid ||
-    dateMoment?._pf?.unusedTokens.length // Stops the user being able to enter a subset of their format;
+    !dateDayjs?.isValid() ||
+    dateDayjs?._pf?.unusedTokens.length // Stops the user being able to enter a subset of their format;
```

Comment:
I'm pretty sure this _pf.unusedTokens.length is not valid dayjs syntax. 

@MonireRasouli can you either point out the area in the dayjs docs where it specifies this if I've missed it, or else work out what this condition does and find an equivalent in dayjs.

Reply:
_pf stores information about the parsed date in moment object. Also it prevents user from entering invalid date format. You are right! _pf is not a valid syntax for dayjs. I don't think if there is any equivalent in dayjs, at least I could not find any.